sdk/healthinsights/azure-healthinsights-cancerprofiling/README.md

```diff
@@ -0,0 +1,263 @@
+            breast, lymph nodes on the right a few areas in the lungs and liver.
+            On biopsy from the lesion in the right breast- triple negative adenocarcinoma. Genetic
+            testing has not been done thus far.
+            Genetic counseling- the patient denies a family history of breast, ovary, uterus,
+            and prostate cancer. Her mother has chronic lymphocytic leukemia (CLL).
+            She is planned to undergo genetic tests because the aggressive course of the disease,
+            and her young age.
+            Impression:
+            Stage 4 triple negative breast adenocarcinoma.
+            Could benefit from biological therapy.
+            Different treatment options were explained- the patient wants to get a second opinion."""
+
+patient_document2 = PatientDocument(
+    type=DocumentType.NOTE,
+    id="doc2",
+    content=DocumentContent(source_type=DocumentContentSourceType.INLINE, value=doc_content2),
+    clinical_type=ClinicalDocumentType.PATHOLOGY,
+    language="en",
+    created_date_time=datetime.datetime(2021, 10, 20),
+)
+
+doc_content3 = """
+            PATHOLOGY REPORT
+                                    Clinical Information
+            Ultrasound-guided biopsy; A. 18 mm mass; most likely diagnosis based on imaging:  IDC
+                                        Diagnosis
+            A.  BREAST, LEFT AT 2:00 4 CM FN; ULTRASOUND-GUIDED NEEDLE CORE BIOPSIES:
+            - Invasive carcinoma of no special type (invasive ductal carcinoma), grade 1
+            Nottingham histologic grade:  1/3 (tubules 2; nuclear grade 2; mitotic rate 1;
+            total score; 5/9)
+            Fragments involved by invasive carcinoma:  2
+            Largest measurement of invasive carcinoma on a single fragment:  7 mm
+            Ductal carcinoma in situ (DCIS):  Present
+            Architectural pattern:  Cribriform
+            Nuclear grade:  2-
+                            -intermediate
+            Necrosis:  Not identified
+            Fragments involved by DCIS:  1
+            Largest measurement of DCIS on a single fragment:  Span 2 mm
+            Microcalcifications:  Present in benign breast tissue and invasive carcinoma
+            Blocks with invasive carcinoma:  A1
+            Special studies: Pending"""
+
+patient_document3 = PatientDocument(
+    type=DocumentType.NOTE,
+    id="doc3",
+    content=DocumentContent(source_type=DocumentContentSourceType.INLINE, value=doc_content3),
+    clinical_type=ClinicalDocumentType.PATHOLOGY,
+    language="en",
+    created_date_time=datetime.datetime(2022, 1, 1),
+)
+
+patient_doc_list = [patient_document1, patient_document2, patient_document3]
+patient1.data = patient_doc_list
+configuration = OncoPhenotypeModelConfiguration(include_evidence=True)
+cancer_profiling_data = OncoPhenotypeData(patients=[patient1], configuration=configuration)
+
+poller = await cancer_profiling_client.begin_infer_cancer_profile(cancer_profiling_data)
```

Comment:
this is async code, it should instead be 

```python
poller = cancer_profiling_client.begin_infer_cancer_profile(cancer_profiling_data)
result = poller.result()
```

It would then be good to show some print statements on the result so users get a feel for where to find the most important attributes on the response model, something like `print(response.name)`, `print(response.tumor_location)` etc

Reply:
The entire sample is under samples folder, anyway I copy/paste the entire code.